script/Arcueid/Day 10/10_00_ARC10_5E.txt

```diff
@@ -423,32 +421,32 @@ Arcueid's will to fight vanishes.
 [sha:30abc5eecbd659959a22aceedcd895255a37afef]{
 -- Page 653, Offset 26013.
 -- いや……あれは恐れ、だろうか。
-No... perhaps it was more like her fear.
+No... perhaps it's more like she's afraid.
 }
 [sha:e9869b65d6dafbda68fc093d3f9f3fe121d79a8a]{
 -- Page 654, Offset 26014.
 -- 「ようやく自覚したか。このガキの血を吸って殺そうとしたんだよ、アンタは。
-"So you're finally aware. You were gonna suck this brat's blood and kill him, weren't you?
+"%{i}Finally%{/i} you realize. Hope you get you were 'bout to suck his blood and kill 'im, right?
```

Comment:
'im feels excessive imo

Reply:
There are certainly cases where it fits with the flow of how he speaks, but I guess it might be.

Reply:
remove hope.